js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/useFetch.js

```diff
@@ -0,0 +1,20 @@
+/* global arangoFetch */
```

Comment:
Not immediately important, but this could potentially be replaced by the already existing functions in `arangoClient.ts`.

Reply:
I created a separate [ticket](https://arangodb.atlassian.net/browse/FE-213)